raiden/storage/serialization/fields.py

```diff
@@ -11,24 +11,14 @@
 from raiden.utils.typing import Address, Any, ChainID, ChannelID, Optional, Tuple
 
 
-class IntegerToStringField(marshmallow.fields.Field):
-    def _serialize(self, value: int, attr: Any, obj: Any, **kwargs: Any) -> str:
-        return str(value)
+class IntegerToStringField(marshmallow.fields.Integer):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(as_string=True, **kwargs)
 
-    def _deserialize(self, value: str, attr: Any, data: Any, **kwargs: Any) -> int:
-        return int(value)
 
-
-class OptionalIntegerToStringField(marshmallow.fields.Field):
-    def _serialize(self, value: Optional[int], attr: Any, obj: Any, **kwargs: Any) -> str:
-        if value is None:
-            return ""
-        return str(value)
-
-    def _deserialize(self, value: str, attr: Any, data: Any, **kwargs: Any) -> Optional[int]:
-        if value == "":
-            return None
-        return int(value)
+class OptionalIntegerToStringField(marshmallow.fields.Integer):
+    def __init__(self, **kwargs: Any) -> None:
+        super().__init__(as_string=True, required=False, **kwargs)
```

Comment:
note: This change broke DB compatibility and older DBs won't be readable due to the way the block numbers are serialized/deserialized.
In the past `''` was okay for no number but now that is no longer the case.